crypto/rsa/rsa_lib.c

```diff
@@ -50,6 +53,17 @@ int RSA_set_method(RSA *rsa, const RSA_METHOD *meth)
 }
 
 RSA *RSA_new_method(ENGINE *engine)
+{
+    return rsa_new_intern(engine, NULL);
+}
+#endif
+
+RSA *rsa_new_with_ctx(OPENSSL_CTX *libctx)
+{
+    return rsa_new_intern(NULL, libctx);
+}
+
+static RSA *rsa_new_intern(ENGINE *engine, OPENSSL_CTX *libctx)
```

Comment:
I think we all need to get into the habit of routinely changing calls like `FOOerr(FOO_F_WHATEVER, whatever)` to `FOOerr(0, whatever)` or `ERR_raise(ERR_LIB_FOO, whatever)`

Reply:
See #9496 which has scripts and instructions to do the conversions automatically, including a final "don't look back fix it all" just before the first (alpha?) release.

Reply:
Yeah, noted.  However, nothing stops us from doing the small scale change when we stumble upon it, 'cause otherwise, we often end up with Travis failures because the `_F_` macro used no longer matches the function name.  This is one of those.